dist/Makefile

```diff
@@ -1,15 +1,15 @@
 HOSTPORT = 12345
 IMAGENAME = "malibu"
 
-.PHONY: docker run-docker
+.PHONY: docker docker-run docker-run-%
 
 docker:
 	docker build -t $(IMAGENAME) ..
 
-run-docker: docker
+docker-run: docker
 	@echo "Running 'malibu' container on port $(HOSTPORT)"
-	@echo "Open 'http://localhost:$(HOSTPORT)/isbn/suche.html' to access it"
-	docker run --rm -it -p $(HOSTPORT):80 "$(IMAGENAME)"
+	@echo "Open 'http://localhost:$(HOSTPORT)/malibu/isbn/suche.html' to access it"
```

Comment:
Did you test this also on Windows? AFAIK `localhost` is for some reason not working with docker and you have to use the
```
$ docker-machine ip
```
(At the moment I don't think I can test this docker things good, because WiFi is limited here.)